guides/components.md

```diff
@@ -210,23 +210,53 @@ end
 
 After reloading [http://localhost:4000/hello](http://localhost:4000/hello), we should see a very different page, one with no title or CSS styling at all.
 
-To customize the application layout, we invoke a similar function named `put_layout/2`. Let's actually create another layout and render the index template into it. As an example, let's say we had a different layout for the admin section of our application which didn't have the logo image. To do this, copy the existing `app.html.heex` to a new file `admin.html.heex` in the same directory `lib/hello_web/components/layouts`. Then remove everything inside the `<header>...</header>` tags (or change it to whatever you desire) in the new file.
+To customize the application layout, we invoke a similar function named `put_layout/2`. Let's actually create another layout and render the index template into it. As an example, let's say we had a different layout for the admin section of our application which didn't have the logo image. To do this, copy the existing `app` component to a new `admin` component in the `lib/hello_web/components/layouts.ex` module. Then remove everything inside the `<header>...</header>` tags (or change it to whatever you desire) in the new component.
 
-Now, in the `index` action of the controller of `lib/hello_web/controllers/hello_controller.ex`, add the following:
+Now, change the `index` action of the controller of `lib/hello_web/controllers/hello_controller.ex`, to the following:
 
 ```elixir
 def index(conn, _params) do
-  conn
-  |> put_layout(html: :admin)
-  |> render(:index)
+  render(conn, :index)
 end
 ```
 
+And then adjust the `index.html.heex` file to invoke `Layouts.admin` instead of `Layouts.app`:
+
+```heex
+<Layouts.admin flash={@flash}>
+  ...
+</Layouts.admin>
+```
+
 When we load the page, we should be rendering the admin layout without the header (or a custom one that you wrote).
 
-At this point, you may be wondering, why does Phoenix have two layouts?
+By default, Phoenix creates one `root` layout that is rendered for all views and then each view explicitly includes the `app` layout - which is just a regular function component. It is also possible to automatically render the `app` layout in each view. To do this, you can either call `put_view(conn, html: {HelloWeb.Layouts, :app})` or define the `layouts` option when `use`ing `Phoenix.Controller`, which usually happens inside your `hello_web.ex` file:
```

Comment:
I would not mention these layouts in our guides anymore, they just add confusion. We can mention it in Phoenix.Controller docs (which I was planning to do after this is merged, so you can either merge with this, and i move it, or you move it now and i adjust later, any is fine by me).